include/gd/BraceEnabler.hxx

```diff
@@ -0,0 +1,35 @@
+/*
+ *  BraceEnabler.hxx
+ *  Copyright 2021 ItJustWorksTM
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#ifndef GODOT_SMCE_BRACEENABLER_HPP
+#define GODOT_SMCE_BRACEENABLER_HPP
+
+#include <gen/Node.hpp>
+
+namespace godot {
+class BraceEnabler : public Node {
+    GODOT_CLASS(BraceEnabler, Node);
+
+  public:
+    static void _register_methods();
+    void _init();
+    void _notification(int what);
+};
+} // namespace godot
+
```

Comment:
I know ryan made this but make this inherit from Reference instead pls, it does not need to be a node
something like `BraceEnabler.new().enable_braces(editor_node)` would be way better.